cellmlmanip/units.py

```diff
@@ -128,6 +131,9 @@ def __init__(self, store=None):
         # without the prefix to make them unique in the registry.
         self._known_units = set(_CELLML_UNITS)
 
+        # Names of units that are in the cellml spec but we currently do not support.
```

Comment:
I'd put this comment on the definition of _UNSUPPORTED_UNITS and not keep a local copy here - we never add to it after all.